presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotProjectExpressionConverter.java

```diff
@@ -28,30 +33,48 @@
 import com.facebook.presto.spi.relation.RowExpressionVisitor;
 import com.facebook.presto.spi.relation.SpecialFormExpression;
 import com.facebook.presto.spi.relation.VariableReferenceExpression;
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import static com.facebook.presto.pinot.PinotErrorCode.PINOT_UNSUPPORTED_EXPRESSION;
+import static com.facebook.presto.pinot.PinotPushdownUtils.getLiteralAsString;
+import static com.facebook.presto.pinot.query.PinotExpression.derived;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 class PinotProjectExpressionConverter
         implements RowExpressionVisitor<PinotExpression, Map<VariableReferenceExpression, Selection>>
 {
+    private static final Set<String> LOGICAL_BINARY_OPS_FILTER = ImmutableSet.of("=", "<", "<=", ">", ">=", "<>");
```

Comment:
Pinot uses `!=` for not equal, you may need to convert it when generating a pinot query.
Or you can use the map to make it generic just like this `PRESTO_TO_PINOT_OPERATORS`.

Reply:
my understanding here is that based on the calcite spec, `<>` and `!=` are both okay with `!=` permissible based on the level of conformance, which is currently at BABEL. Tested `<>` and this runs well, so curious where this is defined internally?